### ISSUE TITLE
Validation error message is not announced by screen readers - The email opt-in modal

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1780]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker 
Screenshot: 

<img width="617" height="282" alt="Image" src="https://github.com/user-attachments/assets/493c6e99-181c-4661-b550-613c9c873cc5" />

WCAG S.C: 4.1.3 Status Messages (AA)

Issue Description: When an invalid email address is submitted, the visual validation message "Enter a valid email address." is displayed below the email field. However, the error message is not automatically announced by the screen reader. Instead, focus remains on the Subscribe button, and users relying on assistive technologies may not be aware that a validation error has occurred.

**Comment by amberhinds** [comment]
I wonder if we need to report this to ActiveCampaign.

**Comment by pattonwebz** [comment]
This is an ActiveCampaign form but we are using a raw form version that does a direct post. I am surprised that this is not an html5 validation message on the field though.

**Comment by amberhinds** [comment]
@pattonwebz there are two ways to embed the form. A "simple" embed that is just a script and a "full" embed. I suspect that we're using the "full" embed which is the full HTML version because there were accessibility issues with the version that loads the form via script from ActiveCampaign. 

I put the [two different code options here](https://3.basecamp.com/3579237/buckets/16288250/documents/10040383402) for you to reference.

I suspect that the best choice here is to code our own fix.
